src/dht/dht_router.cc

```diff
@@ -28,9 +28,7 @@ HashString DhtRouter::zero_id;
 
 DhtRouter::DhtRouter(const Object& cache, const rak::socket_address* sa) :
   DhtNode(zero_id, sa),  // actual ID is set later
-  m_server(this),
```

Comment:
Requires `this`/

Reply:
interesting... Wonder why this was removed.